dashboard/lib/projects_list.rb

```diff
@@ -215,12 +215,15 @@ def project_and_featured_project_and_user_fields
 
     def fetch_featured_projects_by_type(project_type)
       storage_apps = "#{CDO.pegasus_db_name}__storage_apps".to_sym
-      user_storage_ids = "#{CDO.pegasus_db_name}__user_storage_ids".to_sym
+
+      storage_id_table_name = DCDO.get('user_storage_ids_in_dashboard', false) ? "#{CDO.dashboard_db_name}__user_project_storage_ids" : "#{CDO.pegasus_db_name}__user_storage_ids"
```

Comment:
It might be worth creating a function that returns the storage id table name in storage_id.rb and calling it from all three places.  Up to you since this is largely scaffolding that should be short-lived.